README.md

```diff
@@ -226,14 +226,112 @@ You can take a look at `rpc-call-examples` directory. Please note the following:
   with an updated nonce using the
   [provided python script](https://github.com/sayajin-labs/kakarot/blob/main/scripts/utils/kakarot.py#L273).
 
+## Testing
+
+### Hive
+
+The [Hive] end-to-end test suite is set up in the Github Continuous Integration (CI)
+flow of the repository. This ensures a safe guard when modifying the current RPC
+implementation and/or the [execution layer](https://github.com/kkrt-labs/kakarot).
+
+Due to the current existing differences between the Kakarot EVM implementation which
+aims to be a type 2 ZK-EVM (see the blog post from [Vitalik](https://vitalik.eth.limo/general/2022/08/04/zkevm.html)
+for more details), some of the Hive tests need to be skipped or slightly modified
+in order to pass.
+
+For the [hive rpc tests](https://github.com/kkrt-labs/hive/tree/master/simulators/ethereum/rpc),
+all the websocket related tests are skipped as websocket aren't currently supported
```

Comment:
```suggestion
all the websockets related tests are skipped as websockets aren't currently supported
```

Reply:
I wrote `websockets` but trunk was telling me the word doesn't exist :/